### PR TITLE
code cleanup

### PR DESCRIPTION
**Pull request by fberst**
- use of [[maybe_unused]]

### CODE REVIEW

File: src/Device/Driver/IMI/Protocol/Conversion.cpp [hunk 1]
```diff
@@ -31,9 +31,9 @@ Copyright_License {
 #include <stringapiset.h>
 #endif
 
-#define IMI_SECONDS_IN_MINUTE       (60)
-#define IMI_SECONDS_IN_HOUR      (60*60)
-#define IMI_SECONDS_IN_DAY    (24*60*60)
+static constexpr int IMI_SECONDS_IN_MINUTE = 60;
+static constexpr int IMI_SECONDS_IN_HOUR = 60*60;
+static constexpr int IMI_SECONDS_IN_DAY = 24*60*60;
```
Comment (MaxKellermann) [inline src/Device/Driver/IMI/Protocol/Conversion.cpp:36]: Wrong type. This shouldn't be signed.

**Comment by MaxKellermann** [comment]
btw. next stop: finally enable `-Wunused` ;-)

**Comment by fberst** [comment]
> btw. next stop: finally enable `-Wunused` ;-)

I will have a look at it :-)
Isn't this already enabled because it's included in -Wall?

**Comment by MaxKellermann** [comment]
> I will have a look at it :-) Isn't this already enabled because it's included in -Wall?

Unfortunately not.
https://github.com/XCSoar/XCSoar/blob/dfeecc7390225abbdc90720e28fa933e22e7b82c/build/warnings.mk#L16-L17

File: src/Device/Driver/IMI/Protocol/Conversion.cpp (hunk 1 above)
Comment (fberst) [reply src/Device/Driver/IMI/Protocol/Conversion.cpp]: i changed the types to unsigned

**Comment by groundhogxc** [comment]
I guess my comment would have been better put here:

Would it be possible to please combine all these basically identical (and fairly trivial) minor file changes into big combined commits with a common description? The commit list becomes extremely cluttered in this way, and it is cumbersome to find other things - and these particular commits and their comments are, while probably good and important, not extremely informative when reading the descriptions :-)

But thanks for your work either way!

**Comment by MaxKellermann** [comment]
> Would it be possible to please combine all these basically identical (and fairly trivial) minor file changes into big combined commits with a common description?

Agree. Splitting into so many micro-commits is not useful.

**Comment by fberst** [comment]
ok, i will combine the changes into bigger commits next time

**Pull request merged**